js/languages/en_US.json

```diff
@@ -2345,11 +2345,12 @@
     "UNITED_ARAB_EMIRATES": "United Arab Emirates (The)",
     "UNITED_KINGDOM": "United Kingdom of Great Britain and Northern Ireland (The)",
     "UNITED_STATES": "United States of America (The)",
+    "UNITED_STATES_MINOR_ISLANDS": "United States Minor Outlying Islands",
     "URUGUAY": "Uruguay",
     "UZBEKISTAN": "Uzbekistan",
     "VANUATU": "Vanuatu",
     "VENEZUELA": "Venezuela (Bolivarian Republic Of)",
-    "VIETNAM": "Vietnam",
+    "VIETNAM": "Việt Nam",
```

Comment:
Should this really be localized like this? The other countries are in English and it is up to the translation files to localize.

Reply:
I saw it that way on the ISO site I was checking the data against. But checking elsewhere it should be left as the English version. I'll update it.